_topic_map.yml

```diff
@@ -1450,7 +1450,7 @@ Topics:
   - Name: Recovering from expired control plane certificates
     File: scenario-3-expired-certs
 ---
-Name: Migration
+Name: Migration Tookit for Containers
```

Comment:
@apinnick @vikram-redhat Just wondering, was this change to the bucket name discussed?

There is migration planning information under here that is not specific to the toolkit, and if I understand correctly, the toolkit is CAM, where there was also the CPMA tool for migrating the control plane in this section.

So I'm just curious why "Migration" was no longer sufficient as a heading here.

Reply:
@bergerhoffer that is a good point. Hmm.. might need to confirm with Audrey and Avital. I assumed that this was discussed from that team's perspective and the change acked and approved.

Reply:
All the migration projects were rebranded as "Migration Toolkit for ... Containers/Virtualization/Applications."

https://docs.google.com/presentation/d/1ICPnUDhP2oBInRp7wEDKnR7Qn31cI2C441zRrbQZPG4/edit#slide=id.g6191ab6f91_1_15

Reply:
@apinnick Sure, but the overall heading for this section was always just "Migration" and not specifically "CAM". Did you see any issues with keeping it as just "Migration"?

Reply:
@bergerhoffer  If the migration planning docs ever include non-MTC migration tools, then they should be broken out and  placed in a general "Migration" section.

If the migration planning docs are never going to include non-MTC migration tools (CPMA doesn't really count because it's a one-off and near EOL), I am not sure what the advantage would be in calling them "Migration". 

Currently, the planning docs cover the differences between OCP 3 and 4 and migration considerations. MTC needs its own guide because it is a branded 'solution' **and** it needs the planning sections.  Are you recommending that we leave the Migration bucket as is, add an MTC bucket, and use the planning modules in both buckets?

Reply:
I'm not sure what I'd recommend because I wasn't aware that this change was coming or what the motivations behind it were. I just noticed the change yesterday when trying to link someone to the planning section. The change surprised me because I figured that people might be looking for general migration information without toolkit-specifics. But if we don't think that people will be looking for migration information without using the toolkit, then it's fine. Again, I just wanted to make sure that the core OCP side was considered as well, since I hadn't heard that this change was coming.

Reply:
@bergerhoffer This [presentation](https://docs.google.com/presentation/d/1ICPnUDhP2oBInRp7wEDKnR7Qn31cI2C441zRrbQZPG4/edit#slide=id.g6191ab6f91_1_15) might make it clearer. All the migration tools were rebranded to unify their names. Project Bombay includes CAM, which was the tool for OpenShift migration, so that's why it became MTC. Currently, there is no downstream OpenShift migration tool apart from CAM/MTC. That's why the section was renamed. 

The actual file structure was not changed. The planning section URL has not changed. 
Only the titles were changed in the topic map and in the modules. Unless our procedures are telling users to navigate to docs.openshift.com and click Migration > Planning your migration, this change should not be a problem. The links should still work.